export/pdf-component.js

```diff
@@ -144,7 +149,7 @@ class PrintToPdf extends HTMLElement {
 
       const anchor = document.createElement('a');
       anchor.href = finalPdf;
-      anchor.download = event.detail?.fileName || 'file.pdf';
+      anchor.download = `${event.detail?.fileName}.pdf` || 'file.pdf';
```

Comment:
I personally don't think we need to add the extension, the filename should include the extension. If you really want to do it, we will need to change the code on the memberhub to prevent a .pdf.pdf filename. (Windows should be able to open the file fine, but it looks nasty) 
![image](https://user-images.githubusercontent.com/1942073/180638443-ea2bfbfe-9d16-465e-8213-9f66cc8dbb82.png)

Reply:
Got your point, added the extension because the library it is to generate/download PDF so doesn't make so much sense if we pass the extension from outside right? I think we can fix the name in memberhub as well.